src/app/public/modules/format/fixtures/format.component.fixture.html

```diff
@@ -16,7 +16,7 @@
     (click)="testClick()"
   >
     One
-  </button>
+  </button>&nbsp;
```

Comment:
This is needed because of changes made to `sky-label`. See: https://github.com/blackbaud/skyux-indicators/pull/77